R/export.R

```diff
@@ -70,7 +76,7 @@ export <- function(
   base_files <- c(shinylive_common_files("base"), shinylive_common_files("r"))
   if (verbose) {
     p <- progress::progress_bar$new(
-      format = "[:bar] :percent",
+      format = "[:bar] :percent\n",
       total = length(base_files),
       clear = TRUE,
       # show_after = 0
```

Comment:
If we're adding `\n` (which makes a new line for each update), we should not clear the last message and we should show every message at the beginning

```suggestion
      clear = FALSE,
      show_after = 0
```